packages/sonarwhal/src/lib/cli/analyze.ts

```diff
@@ -117,6 +117,16 @@ const setUpUserFeedback = (sonarwhalInstance: Sonarwhal, spinner: ORA) => {
     });
 };
 
+const showMissingAndIncompatiblePackages = (resources: SonarwhalResources) => {
+    if (resources.missing.length > 0) {
```

Comment:
I was thinking about putting that output in the `askUserToInstallDependencies`. Can you move it there?